<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-9E25G1U" failures="2" tests="16" name="com.tutorialsninja.qa.testCases.TestLogin" time="141.300" errors="0" timestamp="2023-08-03T12:54:44 IST" skipped="0">
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationaLoginAfterThatLogoutAndBrowserBackUsingBrowserButtton" time="10.113">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.tutorialsninja.qa.testCases.TestLogin.verifyLoginToApplicationaLoginAfterThatLogoutAndBrowserBackUsingBrowserButtton(TestLogin.java:146)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyLoginToApplicationaLoginAfterThatLogoutAndBrowserBackUsingBrowserButtton -->
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationWithMultipleTimeUnsuccessfulLoginAttempt" time="6.389"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationWithMultipleTimeUnsuccessfulLoginAttempt" time="8.413"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationWithMultipleTimeUnsuccessfulLoginAttempt" time="6.917"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationWithMultipleTimeUnsuccessfulLoginAttempt" time="6.143"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationWithMultipleTimeUnsuccessfulLoginAttempt" time="7.780"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationWithMultipleTimeUnsuccessfulLoginAttempt" time="6.676"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationWithMultipleTimeUnsuccessfulLoginAttempt" time="10.415"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationWithMultipleTimeUnsuccessfulLoginAttempt" time="6.724"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationWithMultipleTimeUnsuccessfulLoginAttempt" time="6.826"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationWithMultipleTimeUnsuccessfulLoginAttempt" time="12.788"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationByCopyingPasswordFiledText" time="8.887"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationCheckPasswordIsNotVisibleInPageSource" time="6.237">
    <failure message="did not expect [xyzabc123   ] but found [xyzabc123   ]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: did not expect [xyzabc123   ] but found [xyzabc123   ]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failEquals(Assert.java:1417)
at org.testng.Assert.assertNotEqualsImpl(Assert.java:156)
at org.testng.Assert.assertNotEquals(Assert.java:2033)
at org.testng.Assert.assertNotEquals(Assert.java:2055)
at com.tutorialsninja.qa.testCases.TestLogin.verifyLoginToApplicationCheckPasswordIsNotVisibleInPageSource(TestLogin.java:190)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyLoginToApplicationCheckPasswordIsNotVisibleInPageSource -->
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationAfterChangingPassword" time="11.689"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationCheckUserIsAbleToNavigateToOtherPages" time="11.049"/>
  <system-out/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verifyLoginToApplicationAfterChangingPasswordAndChangingBackToOriginalPassword" time="14.254"/>
  <system-out/>
</testsuite> <!-- com.tutorialsninja.qa.testCases.TestLogin -->
